css-three-column-layout/styles.css

```diff
@@ -0,0 +1,68 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  color: white;
+  background-color: rgb(49, 49, 233);
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+}
+
+.container {
+  max-width: 2000px;
```

Comment:
I would reduce the size of this container by at least half. It would make using less margin later much easier!